### DISCUSSION TITLE
How can I get Quickdraw badge??

### DISCUSSION THREAD

**Discussion by thundra1125** [community/community#20645] [Other Feature Feedback, Questions, & Ideas] [answered]
I don't know how I can get Quickdraw badge? please let me know...

**Comment by topstarwebking** [comment, marked as answer]
Hey @Webmaster1125 . Nice to talk you.
If you implement this, you will get this badge.

appear when you create a new PR and they are lost within 5 minutes of opening

Hope this will be help you.

**Reply by The-Able** [reply]
Thanks,

**Reply by The-Able** [reply]
Thanks,

**Reply by migraytech** [reply]
I dont follow

**Reply by nakatsinho** [reply]
Oh... Great! Nice to know...

**Reply by Wazbat** [reply]
> I dont follow

Create and merge a PR within the span of 5 minutes

**Reply by DrMadWill** [reply]
> If you implement this, you will get this badge.

What is this ?

**Reply by Alexei567E** [reply]
.github/workflows/browser-test.yml

**Reply by Adswords** [reply]
keep up the good work.

**Reply by webvs2** [reply]
@Webmaster1125  It's nice talking to you

**Reply by gitblanc** [reply]
really?

**Reply by dunkelhai** [reply]
Thanks :D

**Reply by Undertone0809** [reply]
What does mean lost ?

**Reply by mohsin** [reply]
@Undertone0809 You know, the PR is lost... like you just lose the PR. It's a figure of speech just like how [you fold in the cheese](https://youtu.be/fCVKCUB5w50?t=35) or [write something off](https://youtu.be/aCP27_vquxQ?t=41). 🤷

**Reply by ghost** [reply]
thanks.
^_^

**Reply by Sriansh-raj** [reply]
Here is the public repo on list of available **GitHub achievements** and how to **get them**

Repo: **https://github.com/Sriansh-raj/GitHub-Achievements**

You can star it for your reference 🌟

Happy GitHub :octocat:

❤️

**Reply by sanon8268** [reply]
thanks

**Reply by skanda890** [reply]
Thanks.

**Reply by davidlotfi** [reply]
Thanks bro

**Reply by davidlotfi** [reply]
#thanks

**Reply by davidlotfi** [reply]
<3

**Reply by Kazuma275** [reply]
thanks

**Reply by HammadRashid1997** [reply]
You can create an issue in any of your repositories and close it within 5-minute time frame. I tried it with public repositories, idk if it works with private repositories as well or not.

**Reply by su-bonbon** [reply]
thank you

**Reply by dummyyy12345** [reply]
I initially tested it with a private repository and nothing showed up, but when I switched to a public repository, it appeared! So, it looks like it only works for public repositories.

**Reply by deep-explorer** [reply]
How to get quickdraw badge ? Anyone to explain in details?

**Reply by icaroab** [reply]
To earn the Quick Draw badge on GitHub, you need to contribute to a project in the "Arts & Design" category on GitHub. The Quick Draw badge is part of the GitHub profile's "Arts & Design" section, which showcases contributions to repositories related to art, design, or creative projects.

Here's how you can earn the Quick Draw badge:

Contribute to Art-related Projects: Look for repositories on GitHub that are related to art, design, or creative projects. This could include repositories for graphic design, illustration, animation, 3D modeling, and more.

Make Meaningful Contributions: Once you've found a project that interests you, contribute in a meaningful way. This could involve creating new artwork, improving existing designs, fixing bugs, adding documentation, or any other contribution that adds value to the project.

Submit Pull Requests: After making your contributions, submit pull requests to the repository maintainers. Ensure that your contributions follow the project's guidelines and standards.

Collaborate with the Community: Engage with the project maintainers and other contributors. Discuss your contributions, provide feedback, and collaborate with others to improve the project.

Earn the Badge: Once your contributions are accepted and merged into the project, GitHub will recognize your contributions in the "Arts & Design" section of your profile, and you'll earn the Quick Draw badge.

Remember, earning badges on GitHub is not just about collecting achievements but also about actively participating in open-source communities, learning from others, and contributing to projects that you're passionate about. So, focus on making meaningful contributions and engaging with the community.

**Reply by deep-explorer** [reply]
Thanks bro.

**Reply by Kazuma275** [reply]
Hey

**Comment by Wazbat** [comment]
Create and merge a PR within the span of 5 minutes

**Reply by markkdamm** [reply]
I did it yet I haven't received a badge

**Reply by Wazbat** [reply]
> I did it yet I haven't received a badge

I looked through your history and found one PR you made around the same time as this comment, rohya8/mean-stack-interview-questions#2

The PR was not merged, only created. It must be created and merged in a 5 minute window

**Reply by SanjeebLama** [reply]
> I did it yet I haven't received a badge

Simply create a PR and close it. 
You don't even have to merge it.
I just tried and it worked.

**Reply by massimo1220** [reply]
thanks, @SanjeebLama

**Reply by elywin** [reply]
> > I did it yet I haven't received a badge
> 
> Simply create a PR and close it. You don't even have to merge it. I just tried and it worked.

it worked

**Reply by anekemichael** [reply]
Thanks Guys.. It worked for me

**Reply by rudrakshkarpe** [reply]
I did the same but I can't see the badge on my profile, how long does it take to get the badge?

**Reply by KWojcik243** [reply]
Yea, you can create PR and just close it.

**Reply by Alpha2Chris14** [reply]
Thanks

**Reply by PrathibhaSathyanjalee** [reply]
> I did the same but I can't see the badge on my profile, how long does it take to get the badge?

same

**Reply by Jayantkhandebharad** [reply]
how long it takes for badge

**Reply by danydin** [reply]
same issue!

**Reply by Iano-theDev** [reply]
should not take more than 10 minutes

**Reply by davidlotfi** [reply]
❤

**Reply by SurajjBhardwaj** [reply]
issue will also work the same, create an issue and then close it within 5 mins.

**Reply by Kazuma275** [reply]
thanks

**Reply by su-bonbon** [reply]
thank you

**Reply by Mrhuman02** [reply]
Gshs

**Reply by Kazuma275** [reply]
Hey

**Comment by thundra1125** [comment]
I will make you let know.

**Reply by abhisek247767** [reply]
simple create a new repository then create new issue and closed it within 5 minutes. and congratulation you got the badge. also you do also PR.

**Reply by versatility519** [reply]
within 5 mins?

**Reply by jingeno** [reply]
@versatility519 Correct...the whole point of "quick draw" is that it was done quickly.

**Comment by hafisn07** [comment]
Try to Close an issue or a pull request within 5 min of opening

**Reply by Jayantkhandebharad** [reply]
how long it takes to get badge on profile?

**Reply by danydin** [reply]
same issue!

**Comment by Potential17** [comment]


**Reply by hafisn07** [reply]
You have to make a pull request and the repo owner should be pulled and merge the request within a span of 5 minutes.
(Remember that you must ensure that the repo is forked to your profile and cloned)

**Reply by HAMMAS-SALEEM** [reply]
I guess there is a deadline for each badge and for quick draw badge we have crossed the deadline.

**Reply by Wazbat** [reply]
It doesn't have to be a fork. I did it in a private org repo directly. No fork involved

**Comment by als-creator** [comment]
Pair Extraordinaire
who can help me get this achievement? how to do it?

**Reply by UKVeteran** [reply]
When committing to a repo add coauthors in the Description section using:
Co-authored-by: name <name@example.com>
Co-authored-by: another-name <another-name@example.com>

After that create a PR and merge as usual. This worked for me.

**Comment by Echo7659** [comment]
Must be created and merged within 5 minutes

**Reply by Cyronw** [reply]
Don't work for me

**Comment by 617dev** [comment]
i just created a PR and merged it within 5 minutes and i did not receive this badge 😢 does it have to be a public repo or something?

**Reply by Cyronw** [reply]
Same

**Reply by artiomtz** [reply]
1. You get it if you close a PR/issue within 5 minutes. The PR has to be successfully merged.
2. Make sure you have "Contributions & Activity" set to "Include private contributions on my profile" in the Settings > Public profile.
3. Click the Achievements title on your profile page. If it's already in Achievements but not shown, open the achievement and toggle the eye icon for it to be shown on the profile page.

**Comment by UKVeteran** [comment]
Create a PR on a repo and then merge it. Worked for me.

**Reply by massimo1220** [reply]
good work, @UKVeteran

**Reply by aleebaba3594** [reply]
really?

**Reply by ranjiGT** [reply]
Yes you can open a issue within your repo and close it under 5 mins. It works!

**Comment by thundra1125** [comment]
OK, I will teach.

**Reply by sahana-github** [reply]
HI @dennis1125 Created a PR repo and merged it within 5 minutes of time but no badge 😐

**Reply by Iano-theDev** [reply]
Hi @sahana-github, if merging within 5 minutes does not work. Try this, create a pull request and cancel it immediately after it has been created. You'll get the badge after a while ....

**Reply by sahana-github** [reply]
@dennis1125 thanks this worked i got a badge hehe 😁

**Reply by abhisek247767** [reply]
then may be you already have that badge

**Comment by ter1203** [comment]
Created a PR on a public repo and merged it within 5 minutes of time, but still does not work.
Tried more than 10 times... failed every time.
Please help me.

**Comment by aleebaba3594** [comment]
Create a PR on public a repo and merge it all are saying that....but did not worked for me

**Reply by AyushJain001** [reply]
should it be on our own repository or others one

**Reply by anekemichael** [reply]
> should it be on our own repository or others one

Yeah.. Sure I can be your own Repo

**Reply by matinrasooli** [reply]
aaaaaaaaaaaaaaaaaaaaaaa

**Reply by Adesoji1** [reply]
it didnt work for me either

**Comment by codeperfectplus** [comment]
I just got the badge accidentally.

**Reply by nakatsinho** [reply]
Lol

**Comment by marcosbaradelli** [comment]
Try to close an issue or pull request within 5 minutes of opening it, it's the quickest and easiest thing to do.

**Comment by thundra1125** [comment]
Let me explain...

**Comment by Cyronw** [comment]
Guys i can't get it too help pls

**Comment by nffdev** [comment]
How to get this github badge cuz it doesn't working for me..

**Comment by eziosudo** [comment]
I created a PR and close it immediate by mistake, then I get Quickdraw badge, that's funny an accident brings me a badge.

**Comment by Faiqlatheef** [comment]
After answering two discussions, You will get Galaxy Brain badge
And Add 2 or more branches and create pull request and pull it You will get one
More details here:
https://github.com/Schweinepriester/github-profile-achievements

**Comment by olivermen** [comment]
Can I get Quickdraw badge twice? like Pull Shark.

**Reply by Iano-theDev** [reply]
no, I dont think so

**Comment by Do-hyun-Kim** [comment]
You must Pull Request and merge within 5 minutes to receive it

**Comment by arghyadutta080** [comment]
It's a simplest trick, which worked for me. 
Just create a Pull Request in any of your own public repository ( my suggestion : profile readme.md ). And close that Pull Request within 5 minutes. 
The badge will be available in your profile immediately after that.

**Comment by danielmarv** [comment]
Hey @Webmaster1125 . Nice to talk you.
If you implement this, you will get this badge.

appear when you create a new PR and they are lost within 5 minutes of opening

**Comment by kur0neko** [comment]
Make a pull request, and add a viewer to the pull request. Then close pull request within 5 mins you should get the badge.

**Comment by MatthysDev** [comment]
You need to close pr fast

**Comment by v5tech** [comment]
appear when you create a new PR and they are lost within 5 minutes of opening

**Comment by ThePurplePersonxD** [comment]
Can anybody help me?
roblox username: thepurplepersonxD

**Comment by HusanDev7** [comment]
respons #1

**Reply by icaroab** [reply]
Absolutely agree! Enhanced support for Projects would be a game-changer. 🚀

**Comment by HusanDev7** [comment]
responst #2

**Reply by icaroab** [reply]
awesome!

**Comment by Rosellines** [comment]
https://github.com/orgs/community/discussions/176355

**Comment by etsibeko-dev** [comment]
Have a public repo. In your repo you'll have a main, master or whatever branch. Create a 2nd branch, edit one file. Just a minor tweak like replace a certain word in the Readme with a synonym. After that, save, compare, and create a PR on the main branch - the default one. Don't merge the PR. Just close it immediately within 5 minutes.

BTW, I know you already the badge and the question was years ago. But, to anyone who will come across this answer. Just know that I did what I said above, and got my Quickdraw badge immediately.

_Happy collaboration. ❤️_
